pkg/providers/amifamily/bootstrap/eksbootstrap.go

```diff
@@ -78,6 +78,9 @@ func (e EKS) eksBootstrapScript() string {
 	if args := e.kubeletExtraArgs(); len(args) > 0 {
 		userData.WriteString(fmt.Sprintf(" \\\n--kubelet-extra-args '%s'", strings.Join(args, " ")))
 	}
+	if e.InstanceStoreConfiguration != nil {
+		userData.WriteString(fmt.Sprintf(" \\\n--local-disks %s", *e.InstanceStoreConfiguration))
```

Comment:
Reading through the [EKS AMI docs](https://github.com/awslabs/amazon-eks-ami/blob/master/doc/USER_GUIDE.md), I think this needs to be one of "raid0" or "mount". From looking over this code change, it looks like this is just going to be set to one of the enum values that we have declared as part of our API

Reply:
Whoops, previously had these enum values as`raid0`/`mount`. Fixed in https://github.com/aws/karpenter/commit/5cc1de801310e130bf6332da9ac71d94f4dd9228